src/lib/services/regions-service.js

```diff
@@ -0,0 +1,10 @@
+const regionsRepo = require('../connectors/repos/regions');
+const { NotFoundError } = require('../errors');
+
+const getRegionCode = async regionId => {
```

Comment:
It might be more flexible for the future to be able to get a Region service model - think the mapping code already exists:
```
const getRegion = async(regionId) => {
  const region = await regionsRepo.findOne(regionId);
  if (region) return mappers.region.dbToModel(region);
  throw new NotFoundError(`Region ${regionId} not found`);
}
```

The consuming code can be updated to pull out the region ID